.github/workflows/knative-dependabot.yaml

```diff
@@ -0,0 +1,36 @@
+name: depndabot - update-deps.sh
+
+on:
+  pull_request:
+
+permissions:
+  contents: write
+
+jobs:
+  generate_some_code:
+    name: update deps
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+          path: ./src/github.com/${{ github.repository }}
+          fetch-depth: 0
+
+      - name: Run ./hack/update-deps.sh
+        working-directory: ./src/github.com/${{ github.repository }}
+        run: ./hack/update-deps.sh
```

Comment:
You'll want to run `update-codegen` as well

Reply:
Okay, `update-codegen.sh` includes `update-deps.sh` so replaced with `update-codegen.sh`

Reply:
We'll want to run `update-codegen.sh` _after_ `update-deps.sh` - updates in dependencies could change the generated code (ie. generating clients)

Reply:
Okay, updated. Just note, the client codes (`k8s.io/*` and `knative.dev/*` are ignored by dependabot so it is not supposed to be bumped automatically.